src/clang-cpp-frontend/clang_cpp_adjust.h

```diff
@@ -32,6 +41,37 @@ class clang_cpp_adjust : public clang_c_adjust
   void adjust_side_effect(side_effect_exprt &expr) override;
   void adjust_new(exprt &expr);
   void adjust_struct_method_call(member_exprt &expr);
+
+  /**
+   * methods for implicit GOTO code generation
+   */
+  void gen_vptr_initializations(symbolt &symbol);
+  /*
+   * generate vptr initialization code for constructor:
+   *  this->VptrBLAH@BLAH = $vtable::BLAH
```

Comment:
What do the multiple `BLAH` stand for in this comment? Do they refer to a function's identifier, some counter for uniqueness, the constructor's class's name or what?

Reply:
Good point. I've added comment to explain the meaning of 'BLAH' - constructor's class's name